nsv13/code/game/objects/items/munitions_items.dm

```diff
@@ -40,6 +40,11 @@
 	channels = list(RADIO_CHANNEL_MUNITIONS = 1, RADIO_CHANNEL_SUPPLY = 1)
 	independent = TRUE
 
+/obj/item/encryptionkey/headset_bridge
+	name = "bridge radio encryption key"
+	icon_state = "com_cypherkey"
+	channels = list(RADIO_CHANNEL_SUPPLY = 1, RADIO_CHANNEL_SERVICE = 1, RADIO_CHANNEL_COMMAND = 1)

```

Comment:
This is completely identical to the XO key as it stands. Does this really justify a new type to begin with?

Reply:
It doesn't need a new type and this is not a munitions item so would not be in this file.

Reply:
The idea was that the bridge officer isn't given the HoP encryption key if they ever needed a new one, but yeah, it's probably useless.